apps/webapp/app/components/primitives/NamedIcon.tsx

```diff
@@ -247,3 +254,11 @@ export function NamedIconInBox({
     </div>
   );
 }
+
+export function TablerIcon({ name, className }: { name: string; className?: string }) {
+  return (
+    <svg className={className}>
+      <use xlinkHref={`/icons/tabler-sprite.svg#${name}`} />
```

Comment:
@ericallam should this be a path imported from the app folder? We should cache the svg because it's quite big